modules/theme/src/themes/default/collections/menu.variables

```diff
@@ -55,7 +55,7 @@
 @appHeaderSecondaryPanelPortalSwitchesLeftSpacing: 14px;
 
 @appHeaderSecondaryPanelPortalSwitchBackground: inherit;
-@appHeaderSecondaryPanelPortalSwitchColor: @mutedTextColor;
+@appHeaderSecondaryPanelPortalSwitchColor: @lightTextColor;
```

Comment:
Let's keep the font colors as it is and try to find out the reason for the thick border.
I think the background in the header is peaking in. Please check @yasinmiran 

![image (2)](https://user-images.githubusercontent.com/25959096/108677154-46570780-750f-11eb-857a-c0087244b0f5.png)

Reply:
This is the result of the revert. `@lightTextColor` is the previous value.